code/src/Components/Drink.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+export const Drink = ({ drink, setDrink }) => {
+  const handleDrinkChange = (event) => {
+    setDrink(event.target.value);
+  }
+  return (
+    <div className="outerWrapper">
+      <div className="questionContainer">
+        <img className="questionImage" alt="coffee" src="https://images.unsplash.com/photo-1621135177072-57c9b6242e7a?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=774&q=80" />
+        <p className="question3">What would you like to drink?</p>
+        <div>
+          <div className="inputContainer">
+            <input
+              className="inputRadio"
+              type="radio"
+              id="drinkOne"
+              value="Coffee"
+              onChange={handleDrinkChange}
+              checked={drink === 'Coffee'} /> Coffee
```

Comment:
Here you can use the `<label>` tag for the inputs. This will have the added benefit that you can then also click on the entire label text to select that option, instead of having to hit the small radio input circle. For example:
```
         <label htmlFor="blueberry">
            <input
              type="radio"
              id="blueberry"
              value="Blueberries"
              onChange={handlePancakesChange}
              checked={pancakes === 'Blueberries'} />
              Blueberry please!
          </label>
```